docs/guides/migration-guide-v1.0.0.md

```diff
@@ -118,9 +98,9 @@ resources for your databases. Like so:
   
     cloud_provider {...}
   
-    // Add this block if you want to create a new subscription. 
-    // Skip, if you are importing an existing subscription.
-    // In this block, define your hardware specification for you databases in the cluster.
+   	 // For migrating, you can skip this block,
```

Comment:
The formatting doesn't look right here

Reply:
I have corrected this in this PR #256, so should be resolved once that is merged